client/src/app/interfaces/source-agent-parameter.ts

```diff
@@ -16,4 +16,5 @@ export interface SourceAgentParameter {
   name: string;
   type: string;
   value?: string | number;
+  geo?: boolean;
```

Comment:
I'm not too fond of expanding the interface for just this one feature.
Maybe we can add `geo` as a `type` instead?

Reply:
Good idea, done